lllabo5.cpp

```diff
@@ -0,0 +1,138 @@
+﻿#include <iostream>
+using namespace std;
+
+class Sneakers {
+private:
+    string brand, color, material;
+    int size, price, quantity, numberOfSales;
+public:
+    Sneakers() { brand = ""; color = ""; material = ""; size = 0; price = 0; quantity = 0; numberOfSales = 0; }
+
+    void setBrand(string brand) { this->brand = brand; }
+    void setColor(string color) { this->color = color; }
+    void setMaterial(string material) { this->material = material; }
+    void setSize(int size) { this->size = size; }
+    void setPrice(int price) { this->price = price; }
+    void setQuantity(int quantity) { this->quantity = quantity; }
+    void setNumberOfSales(int numberOfSales) { this->numberOfSales = numberOfSales; }
+
+    string getBrand() { return brand; }
+    string getColor() { return color; }
+    string getMaterial() { return material; }
+    int getSize() { return size; }
+    int getPrice() { return price; }
+    int getQuantity() { return quantity; }
+    int getNumberOfSales() { return numberOfSales; }
```

Comment:
Please separate methods and their budies by creating another .cpp file, where bodies of methods will be